app/assets/stylesheets/components/logo.scss

```diff
@@ -1,5 +1,6 @@
 .logo {
   a {
+    width: 100%;
```

Comment:
oh dang.. no.

I was trying something else that needed it and didn't get it deleted when I reverted. good catch